spec.html

```diff
@@ -20816,14 +21648,14 @@ <h1>Static Semantics: Early Errors</h1>
 
     <emu-clause id="sec-runtime-semantics-instantiateasyncfunctionobject" oldids="sec-async-function-definitions-InstantiateFunctionObject" type="sdo" aoid="InstantiateAsyncFunctionObject">
       <h1>Runtime Semantics: InstantiateAsyncFunctionObject</h1>
-      <p>With parameter _scope_.</p>
+      <p>With parameters _scope_ and _privateScope_.</p>
```

Comment:
There's an existing invocation of InstantiateAsyncFunctionObject that hasn't been given a second argument.

Reply:
Where?

Reply:
Sorry, I think I got mixed up between InstantiateAsyncFunctionObject and InstantiateFunctionObject. There's an invocation of the latter in GlobalDeclarationInstantiation that doesn't have an arg for `_privateScope_`.

Reply:
Fixed in 94b1af23d6ca075c6635bcd5b96c6450ec580f51